include/swaynag/types.h

```diff
@@ -7,6 +7,7 @@ struct swaynag_type {
 	char *font;
 	char *output;
 	uint32_t anchors;
+	int32_t layer;
```

Comment:
Nit: ideally add a small `// enum zwlr_layer_shell_v1_layer or -1 if unset` comment to make it clear what the values can be.